test/util/hca-helpers.js

```diff
@@ -436,12 +436,25 @@ function completeVaInsuranceInformation(client, data, onlyRequiredFields) {
 }
 
 function initApplicationSubmitMock() {
+  // TODO: Fully migrate to v1
+  mock(null, {
```

Comment:
This is so we can support both endpoints.  It would be better to create each endpoint conditionally, but this gets the job done.